ccr/createCCR.php

```diff
@@ -12,26 +12,32 @@
  * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
  */
 
+use OpenEMR\Common\Acl\AclMain;
+use OpenEMR\Common\Session\SessionUtil;
+use OpenEMR\Common\Twig\TwigContainer;
+use PHPMailer\PHPMailer\PHPMailer;
```

Comment:
also generally try to do the use block after globals.php is brought to avoid ever messing up things in that core script. in this case could just put the use for SessionUtil here and the put the other after globals.php is brought in below